pages/linux/create_ap.md

```diff
@@ -4,28 +4,28 @@
 
 - Create an open network with no passphrase:
 
-`create_ap {{wlan0}} {{eth0}} {{MyAccessPoint}}`
+`create_ap {{wlan0}} {{eth0}} {{access_point_ssid}}`
 
 - Use a WPA + WPA2 passphrase:
 
-`create_ap {{wlan0}} {{eth0}} {{MyAccessPoint}} {{MyPassPhrase}}`
+`create_ap {{wlan0}} {{eth0}} {{access_point_ssid}} {{passphrase}}`
 
 - Create an access point without Internet sharing:
 
-`create_ap -n {{wlan0}} {{MyAccessPoint}} {{MyPassPhrase}}`
+`create_ap -n {{wlan0}} {{acces_point_ssid}} {{passphrase}}`
 
 - Create a bridged network with Internet sharing:
 
-`create_ap -m bridge {{wlan0}} {{eth0}} {{MyAccessPoint}} {{MyPassPhrase}}`
+`create_ap -m bridge {{wlan0}} {{eth0}} {{access_point_ssid}} {{passphrase}}`
 
 - Create a bridged network with Internet sharing and a pre-configured bridge interface:
 
-`create_ap -m bridge {{wlan0}} {{br0}} {{MyAccessPoint}} {{MyPassPhrase}}`
+`create_ap -m bridge {{wlan0}} {{br0}} {{access_point_ssid}} {{passphrase}}`
 
 - Create an access port for Internet sharing from the same WiFi interface:
 
-`create_ap {{wlan0}} {{wlan0}} {{MyAccessPoint}} {{MyPassPhrase}}`
+`create_ap {{wlan0}} {{wlan0}} {{access_point_ssid}} {{passphrase}}`
 
 - Choose a different WiFi adapter driver:
 
-`create_ap --driver {{WifiAdapter}} {{wlan0}} {{eth0}} {{MyAccessPoint}} {{MyPassPhrase}}`
+`create_ap --driver {{WifiAdapter}} {{wlan0}} {{eth0}} {{access_point_ssid}} {{passphrase}}`
```

Comment:
Isn't it better if we go for something like this just like we are doing on other tokens?
```suggestion
`create_ap --driver {{wifi_adapter}} {{wlan0}} {{eth0}} {{access_point_ssid}} {{passphrase}}`
```

Reply:
Yes, I'd agree with that. Hadn't even noticed it. 👍

Reply:
Oh, I missed that too! Nice catch :)